build.sbt

```diff
@@ -11,6 +11,8 @@ lazy val root = project("paradox-material-theme-parent", file("."))
     Compile / paradoxNavigationDepth := 3,
     makeSite := makeSite.dependsOn(Compile / paradox).value,
     paradoxMaterialTheme / version := version.value,
+    // this is to avoid triggering update, which will fail
+    Compile / compile := sbt.internal.inc.Analysis.empty,
```

Comment:
@mdedetrich @mkurz Here's my hypothesis on the mysterious behavior. This build uses [intertwined dependency](https://stackoverflow.com/questions/37424513/intertwined-dependencies-between-sbt-plugin-and-projects-within-multi-project-bu) pattern wherein the build itself uses the sbt plugin defined in the build, and specifically the root subproject enables `ParadoxMaterialThemePlugin`.

So while the root subproject does not contain any Scala source, `compile` will trigger an `update` at the root-level, which likely triggers the dependency resolution against theme version 0.6.0-SNAPSHOT, which doesn't exist yet. I don't know if the missing `/` is actually true, or some other red herring bug. In any case, the above demonstrates that short-circuiting the root `Compile / compile` fixes the issue.

Reply:
Thanks for looking into this! I will update the original PR and give you credit for it